src/tags/MethodTag.php

```diff
@@ -1,13 +1,14 @@
-<?php
+<?php declare(strict_types=1);
+
 namespace gossi\docblock\tags;
+
 /**
- * Represents the @method tag.
+ * Represents the "@method" tag.
```

Comment:
Also please use backticks here instead of quotes.

Reply:
Done.